unittests.py

```diff
@@ -19,6 +14,7 @@ def tealiumCallback(self, info, success, error=None):
                                   "tealium_account",
                                   "tealium_profile",
                                   "tealium_environment",
+                                  "tealium_datasource",
                                   "tealium_random",
                                   "event_name",
```

Comment:
Make this change and make a note in the change log, as this could throw anyone (if anyone) has used a previous version of the lib.